content/authentication/troubleshooting-ssh/using-ssh-over-the-https-port.md

```diff
@@ -28,6 +34,12 @@ $ ssh -T -p 443 git@ssh.github.com
 
 If that worked, great! If not, you may need to [follow our troubleshooting guide](/articles/error-permission-denied-publickey).
 
+## Git Client Repository Example
```

Comment:
I don't think this warrants a heading. Instead, you could make the text flow pretty naturally with something like:
> Now you can clone the repository by running the following command:

Reply:
Great suggestion. Updated in dc03859c718c5821be04fc7e169a0f71f663931f.